config/locales/devise.en.yml

```diff
@@ -32,10 +32,16 @@ en:
       success: "Successfully authenticated from %{kind} account."
     passwords:
       no_token: "You can't access this page without coming from a password reset email. If you do come from a password reset email, please make sure you used the full URL provided."
+      nonexisten_email: "Unable to find user with email '%{email}'."
```

Comment:
Are this custom messages? In that case I think it would be better to add them on `en.yml` file instead of `devise.en.yml` so that we leave the one of devise just with the default messages.